### PR TITLE
Add demo links to DataGrid & TreeList API reference

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: api-reference/10 UI Components/GridBase/1 Configuration/showBorders.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ default: false
 Specifies whether the outer borders of the UI component are visible.
 
 ---
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Appearance/"
+}
```
Comment (pavel-2020) [inline api-reference/10 UI Components/GridBase/1 Configuration/showBorders.md]:
```suggestion
#include common-demobutton-named with {
    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Appearance/",
    name: "DataGrid"
}
#include common-demobutton-named with {
    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/Overview/",
    name: "TreeList"
}
```

File: api-reference/10 UI Components/GridBase/1 Configuration/showRowLines.md [hunk 2]
```diff
@@ -8,6 +8,9 @@ default: false, true (iOS, Material)
 Specifies whether horizontal lines that separate one row from another are visible.
 
 ---
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Appearance/"
+}
```
Comment (pavel-2020) [inline api-reference/10 UI Components/GridBase/1 Configuration/showRowLines.md]:
```suggestion
#include common-demobutton-named with {
    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Appearance/",
    name: "DataGrid"
}
#include common-demobutton-named with {
    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/RowEditing/",
    name: "TreeList"
}
```

File: api-reference/10 UI Components/dxTreeList/3 Methods/getNodeByKey(key).md [hunk 3]
```diff
@@ -12,6 +12,10 @@ The [Node](/api-reference/10%20UI%20Components/dxTreeList/4%20Node '/Documentati
 The node's key.
 
 ---
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/LocalReordering/"
```
Comment (AlexJameson) [inline api-reference/10 UI Components/dxTreeList/3 Methods/getNodeByKey(key).md]: Is this related to TreeList?

File: api-reference/10 UI Components/dxDataGrid/3 Methods/getSelectedRowKeys().md [hunk 4]
```diff
@@ -9,6 +9,10 @@ Gets the currently selected rows' keys.
 Keys of currently selected rows or a Promise that is resolved with an array of keys. The keys are stored in the order the user selects rows.
 
 ---
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/CellEditingAndEditingAPI/"
```
Comment (AlexJameson) [inline api-reference/10 UI Components/dxDataGrid/3 Methods/getSelectedRowKeys().md:13]: this demo shows getSelectedRowsData(). Is it ok?

File: api-reference/10 UI Components/dxTreeList/3 Methods/getNodeByKey(key).md (hunk 3 above)
Comment (RomanTsukanov) [reply api-reference/10 UI Components/dxTreeList/3 Methods/getNodeByKey(key).md]: Wrong demo link

```suggestion
    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/LocalReordering/"
```

Comment (AlexJameson) [inline api-reference/10 UI Components/dxTreeList/3 Methods/getNodeByKey(key).md]:
```suggestion
    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/LocalReordering/"
```

**Review by AlexJameson — APPROVED** [review]
